src/Shared/Recruit.Vacancies.Client/Infrastructure/QueryStore/Projections/LiveVacancy/LiveVacancy.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections.LiveVacancy
+{
+    public class LiveVacancy : QueryProjectionBase
+    {
+        public Guid VacancyId { get; set; }
+        
+        public string ApplicationInstructions { get; set; }
+        public string ApplicationUrl { get; set; }
+        public DateTime ClosingDate { get; set; }
+        public string Description { get; set; }
+        public string EmployerContactEmail { get; set; }
+        public string EmployerContactName { get; set; }
+        public string EmployerContactPhone { get; set; }
+        public string EmployerDescription { get; set; }
+        public Address EmployerLocation { get; set; }
+        public string EmployerName { get; set; }
+        public string EmployerWebsiteUrl { get; set; }
+        public int NumberOfPositions { get; set; }
+        public string OutcomeDescription { get; set; }
+        public string ProgrammeId { get; set; }
+        public IEnumerable<Qualification> Qualifications { get; set; }
+        public string ShortDescription { get; set; }
+        public List<string> Skills { get; set; }
```

Comment:
Could this be IEnumerable or Qualifications become List<Qualification>

Reply:
👍 I'll change to IEnumerable